// 指定protobuf的版本，proto3是最新的语法版本
syntax = "proto3";

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}

// python -m grpc_tools.protoc --python_out=. --grpc_python_out=. -I. hello.proto